src/noisepy/seis/zarrstore.py

```diff
@@ -0,0 +1,154 @@
+import datetime
```

Comment:
@nrich20 , can you move this to a separate PR? All of the above is about the txt->csv/json conversion